src/sql/workbench/contrib/notebook/browser/cellViews/textCell.component.ts

```diff
@@ -236,6 +273,11 @@ export class TextCellComponent extends CellView implements OnInit, OnChanges {
 		}
 	}
 
+	getNodeIndex(n) {
```

Comment:
The name here doesn't seem to match what the logic is doing. Indices normally start at 0 and go up - but this seems to be starting with 0 at the end and walking backwards?

Reply:
Gotcha. I guess this is fine (or at least I don't have a better idea). It's definitely not the depth though - that would imply to me something more like depth in the DOM tree. (so walking parents)

But : 

1. I think this should be a generic function in dom.ts
2. Please add a doc comment explaining the functionality

Reply:
I'll create a separate PR and address this in that. Thanks!